apps/aecore/src/aec_tx_pool.erl

```diff
@@ -742,8 +742,9 @@ get_account(AccountKey, {account_trees, AccountsTrees}) ->
 get_account(AccountKey, {block_hash, BlockHash}) ->
     aec_chain:get_account_at_hash(AccountKey, BlockHash).
 
-check_minimum_fee(Tx,_Hash) ->
-    case aetx:fee(aetx_sign:tx(Tx)) >= aec_governance:minimum_tx_fee() of
+check_minimum_fee(Tx0, _Hash) ->
+    Tx = aetx_sign:tx(Tx0),
+    case aetx:fee(Tx) >= (aetx:gas(Tx) * aec_governance:minimum_gas_price()) of
```

Comment:
This is not correct when Tx is `aect_create_tx` or `aect_call_tx` the fee should _not_ cover `Gas * GasPrice` in this case, it should only cover the basic transaction cost (base_gas + tx_size gas)?!

Reply:
@hanssv I updated it. The min fee of contract txs doesn't include the gas for contract execution.